src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,31 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
+			// nonce check if search form submitted
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: f16265d2038e3fbf17fe4264f221c8b3</sub>